src/features/smartSearch/components/filters/EmailHistory/index.tsx

```diff
@@ -130,19 +122,25 @@ const EmailHistory = ({
               <StyledSelect
                 onChange={(e) => {
                   removeKey(['email', 'campaign']);
-                  setEmailSelectScope(e.target.value as EMAIL_SELECT_SCOPE);
+                  setEmailSelectScope(e.target.value as EmailSelectScopeType);
```

Comment:
I think this is fine for now, but in general, whenever you write ` as ` something in typescript, make absolute sure you have no other (sensible) way of solving the problem. Type casting with `as` is a way (among others) of telling typescript you know better than typescript does. That can often be the case, but is likely to lead to bugs now or in the future.